### PR TITLE
Add documentation on Breez API key

### PR DESCRIPTION
**Pull request by hydra-yse**

### CODE REVIEW

File: src/guide/getting_started.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ Integrating Breez SDK - *Liquid* into your application takes just a few minutes.
 - **[Listening to events](/guide/events.md)**
 - **[Adding logging](/guide/logging.md)**
 
+## API Key
+Some methods require the Breez API key to be set in order to work. You can request one for free by filling our form <a target="_blank" href="{{api_key_form_uri}}">here</a>.
```
Comment (ok300) [inline src/guide/getting_started.md:11]: Clicking this link in the generated mdbook gives a 404, as it tries to access `/guide/{{api_key_form_uri}}`.

File: src/guide/send_payment.md [hunk 2]
```diff
@@ -88,21 +88,28 @@ The SDK will also validate that the amount is within the send lightning limits o
 
 <div class="warning">
 <h4>Developer note</h4>
-When paying an invoice generated by another Liquid SDK instance, or any other application which uses the Boltz swapper internally, the payment will fall back to a direct onchain payment. The advantage of this is the payer will spend less on fees, as they are no longer relying on external services to execute the payment. To learn more about this process and how it works in detail, see the Boltz documentation for 
-<a target="_blank" href="https://docs.boltz.exchange/v/api/magic-routing-hints">Magic Routing Hints (MRH)</a>.
+When paying an invoice generated by another Liquid SDK instance, or any other application which uses the Boltz swapper internally, the payment will fall back to a direct onchain payment. 
+The advantage of this is the payer will spend less on fees, as they are no longer relying on external services to execute the payment. 
+Note that this also means a <a target="_blank" href="{{api_key_form_uri}}">Breez API key</a> <b>will be required</b> in order for the payment to be executed.
```
Comment (ok300) [inline src/guide/send_payment.md:93]: Clicking on the "Breez API key" link gives a 404.

File: src/guide/send_payment.md [hunk 3]
```diff
@@ -88,21 +88,28 @@ The SDK will also validate that the amount is within the send lightning limits o
...
+  <h3>Liquid</h3>
+  {{#include ../components/api_key.md}}
+</div>
 When sending via Liquid, a BIP21 URI or Liquid address can be used as the destination. 
 
 If a Liquid address is used, the optional prepare request amount **must** be set. 
 
 If a BIP21 URI is used, either the BIP21 URI amount or optional prepare request amount **must** be set. When both amounts are set, the SDK will prioritize the **request amount** over the BIP21 amount.
 
+**Note:** If a valid Breez API key is not provided, the method will throw an error requiring you to specify one
```
Comment (ok300) [inline src/guide/send_payment.md:111]: Nit: there should be a dot at the end, as it's a full sentence.

**Review by ok300 — COMMENTED** [review]
The variable substitution didn't work for me, even after upgrading `mdbook` and re-installing our `snippets-processor`.

When running `mdbook serve` or `mdbook build` I get this output:
```
2024-09-20 14:47:02 [WARN] (mdbook::preprocess::cmd): The command wasn't found, is the "variables" preprocessor installed?
2024-09-20 14:47:02 [WARN] (mdbook::preprocess::cmd):   Command: mdbook-variables
```

and there is no variable substitution in the new links.

**Comment by ok300** [comment]
> The variable substitution didn't work for me, even after upgrading `mdbook` and re-installing our `snippets-processor`.

After locally installing
```
cargo install mdbook-variables
```

the variable substitution works.

The "Install dependencies" step in the CI workflows file (`.github/workflows/main.yml`) needs a new `cargo install` line with the above, for this to work in the deployed docs.

**Review by roeierez — APPROVED** [review]

**Pull request merged**